### PR TITLE
docs: update commands

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
update docs

**Did you add tests for your changes?**
NO
**If relevant, did you update the documentation?**
YES
**Summary**

update commands.
Add README.md for `@webpack-cli/configtest`.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,12 +55,17 @@ We organize webpack CLI as a multi-package repository using [lerna](https://gith
...
--   [`webpack-cli init`](./packages/init/README.md#webpack-cli-init) - Create a new webpack configuration.
--   [`webpack-cli info`](./packages/info/README.md#webpack-cli-info) - Returns information related to the local environment.
--   [`webpack-cli migrate`](https://www.npmjs.com/package/@webpack-cli/migrate) - Migrate project from one version to another.
--   [`webpack-cli plugin`](./packages/generators#generators) - Initiate new plugin project.
--   [`webpack-cli loader`](./packages/generators#generators) - Initiate new loader project.
--   [`webpack-cli serve`](./packages/serve/README.md#webpack-cli-serve) - Use webpack with a development server that provides live reloading.
+-   `build|bundle|b [entries...] [options]` - Run webpack (default command, can be omitted).
+-   [`configtest|t [config-path]`](./packages/configtest/README.md#webpack-cli-configtest) - Validate a webpack configuration.
+-   `help|h [command] [option]` - Display help for commands and options.
+-   [`init|c [scaffold...] [options]`](./packages/init/README.md#webpack-cli-init) - Create a new webpack configuration.
```
Comment (rishabh3112) [inline README.md:61]:
```suggestion
-   [`init|c [scaffold...] [options]`](./INIT.md#webpack-cli-init) - Create a new webpack configuration.
```

**Review by rishabh3112 — COMMENTED** [review]
A minor update.

**Review by alexander-akait — APPROVED** [review]

**Review by jamesgeorge007 — APPROVED** [review]

File: packages/configtest/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+This package validates a webpack configuration.
+
+## Installation
+
+```bash
+#npm
+npm i -D @webpack-cli/configtest
+
+#yarn
+yarn add @webpack-cli/configtest -D
```
Comment (anshumanv) [inline packages/configtest/README.md:16]:
```suggestion
yarn add -D @webpack-cli/configtest
```

Comment (anshumanv) [reply packages/configtest/README.md:16]: consistency

Comment (snitin315) [reply packages/configtest/README.md]: fixed

**Comment by alexander-akait** [comment]
@snitin315 need rebase too

**Review by alexander-akait — APPROVED** [review]

**Comment by snitin315** [comment]
Thanks for the rebase 👍🏻

**Pull request merged**